src/actions/google/drive/google_drive.ts

```diff
@@ -161,7 +161,21 @@ export class GoogleDriveAction extends Hub.OAuthAction {
           return form
         }
       } catch (e: any) {
-        winston.error("Can not sign in to Google", {webhookId: request.webhookId} )
+        const errorType = getHttpErrorType(e, this.name)
+        let error: Error = errorWith(
+          errorType,
+          `${LOG_PREFIX} ${e.message}`,
+        )
+        const errorObjectKeys: any = []
```

Comment:
I don't follow why we're pulling all of the errorObjectKeys out.  Are we logging this as part of the error log for future reference of what data we could log?

Reply:
Yes exactly, we are assuming what the error object looks like on line 173, but if we are wrong or if there are other keys on `e` then we could add them after this gets called a few times.